### PR TITLE
Make sure /usr/sbin is present in system PATH

### PR DESCRIPTION
**Pull request by npalaska**
Addresses issue https://github.com/distributed-system-analysis/pbench/issues/2383

### CODE REVIEW

**Review by dbutenhof — CHANGES_REQUESTED** [review]

File: agent/profile [hunk 1]
```diff
@@ -3,6 +3,8 @@
 function _pbench_pathins() {
     if [[ -d "${1}" ]] && [[ ":${PATH:-}:" != *":${1}:"* ]]; then
         PATH="${1}${PATH:+:${PATH}}"
+        if [[$PATH != *"/usr/sbin"*]]; then
```
[the remark this answers is not in the document]
Comment (npalaska) [reply agent/profile:6]: > The `_pbench_pathins` function is designed to do exactly what you're adding here: conditionally add a directory that's not already in `PATH`. It's called repeatedly (e.g., line 38/39) so you're unnecessarily duplicating this check; plus doing the check in a manner inconsistent with the existing check at line 4.

Ah yes thanks for pointing out.


> And while `/usr/sbin` happens to satisfy this particular requirement, I wonder if we should have any others, like maybe `/sbin`, which might not also be in a standard `PATH`. I can't think of any really good automated way to figure out what paths we might actually need beyond a normal user `PATH`, but maybe setting `export PATH=/opt/pbench-agent/util-scripts:/opt/pbench-agent/bench-scripts` without anything else and then making sure `pbench-user-benchmark` and `pbench-fio` work would at least be a start.

Yes I did some tests around this like running a pbench-* commands with one of different versions of PATH (removing everything except `/opt/pbench-agent/util-scripts:/opt/pbench-agent/bench-scripts` and then manually adding paths to system PATH) to check which command needs what but then this creates a lot of permutations. I was running it as a root so I expected that default PATH would contain everything we need based on webb's comment on issue 2383. 

So maybe we could do 
_pbench_pathins /usr/sbin
_pbench_pathins /usr/local/sbin
_pbench_pathins /usr/local/bin
_pbench_pathins /usr/sbin
_pbench_pathins /usr/bin
_pbench_pathins /root/bin

to make sure everything is present in system PATH. And I will do some testing with `/sbin`

**Review by portante — CHANGES_REQUESTED** [review]
Like the suggestion of using "pathin" on the list of known paths we need.

File: agent/profile [hunk 2]
```diff
@@ -6,6 +6,13 @@ function _pbench_pathins() {
     fi
 }
 
+_pbench_pathins "/usr/local/sbin"
+_pbench_pathins "/usr/local/bin"
+_pbench_pathins "/usr/sbin"
+_pbench_pathins "/usr/bin"
+_pbench_pathins "/root/bin"
+_pbench_pathins "/sbin"
```
Comment (portante) [inline agent/profile:14]: These last two I am not sure of now that I see them.  I think it opens an unnecessary set of problems.

File: agent/profile [hunk 3]
```diff
@@ -6,6 +6,13 @@ function _pbench_pathins() {
     fi
 }
 
+_pbench_pathins "/usr/local/sbin"
+_pbench_pathins "/usr/local/bin"
```
Comment (portante) [inline agent/profile:10]: I don't these these two should be included.  Does the pbench-agent RPM install anything in these directories?  If not, then we should not include them.  It is up to the administrator of the system to add them to the path explicitly.

**Review by portante — CHANGES_REQUESTED** [review]

File: agent/profile (hunk 3 above)
Comment (npalaska) [reply agent/profile:10]: Yes the pbench-agent rpm doesnt seem to install anything in these 2 directories, I will remove them

File: agent/profile (hunk 2 above)
Comment (npalaska) [reply agent/profile:14]: Yes I did some tests with pbench-agent commands and `/sbin` doesn't seem to be required in the path (but I wasn't fully sure and its possible that I missed something that actually need this directory in the path so I added it thinking we dont want to come across an issue where something is missing from the system PATH)

**Review by portante — APPROVED** [review]

**Review by webbnh — APPROVED** [review]
👍

**Review by dbutenhof — APPROVED** [review]

**Pull request merged**